gitversion.yml

```diff
@@ -0,0 +1,7 @@
+assembly-file-versioning-format: '{Major}.{Minor}.{Patch}.{CommitsSinceVersionSource}'
+branches:
+  development:
+    regex: development      
+    tag: ''
+    is-mainline: true
+    source-branches: []
```

Comment:
Versions will be applied to `SolutioInfo.cs` as follows

```c#
[assembly: AssemblyVersion("9.2.1.0")]
[assembly: AssemblyInformationalVersion("9.2.1-managing-version.1+670.Branch.managing-version.Sha.52eace732928da5053e2f597bf5c42322405dd4a")]
[assembly: AssemblyFileVersion("9.2.1.670")]
```

It's important to note that `[AssemblyVersion]` will be the same for all builds of given release. This way it will be easier to debug with local builds. Without assembly loader complaining about mismatched versions.

The `assembly-file-versioning-format` pattern above is used to feed the file version.

The informational version generated by gitversion includes extensive metadata.